frontend/public/i18next-parser.config.js

```diff
@@ -1,14 +1,22 @@
+const PARSER_SUPPORTED_LOCALES = {
```

Comment:
This file shouldn't be in the `public` folder if it's just the parser config. `public` is for code that is part of the app itself.

Reply:
I'll try to move it. Fingers crossed it builds.